actors/builtin/storage_market/storage_market_actor.go

```diff
@@ -316,7 +323,7 @@ func (a *StorageMarketActor) OnEpochTickEnd(rt Runtime) *adt.EmptyValue {
 	return &adt.EmptyValue{}
 }
 
-func (a *StorageMarketActor) Constructor(rt Runtime) *adt.EmptyValue {
+func (a *StorageMarketActor) Constructor(rt Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
```

Comment:
nit: can this be moved to the top of the file so it matches the rest of the actors?

Reply:
I'm trying to reduce conflicts with @magik6k implementing the state proper